election/views.py

```diff
@@ -14,6 +14,8 @@
 from election.models import Election, Token, Vote
 from libs import majority_judgment as mj
 
```

Comment:
There are useless carriage returns here. Could you run the black formatter to avoid those things?